### DISCUSSION TITLE
SDK version :2.0.0-preview007   CreateAsync() Json Error

### DISCUSSION THREAD

**Discussion by chandra-rayaprol** [Azure/azure-iot-sdk-csharp#3409] [General]
The method IotHubServiceClient.Devices.CreateAsync(new Device("deviceId");
is throwing an exception: 
Newtonsoft.Json.JsonReaderException: Could not convert string to DateTimeOffset: 0001-01-01T00:00:00. Path 'connectionStateUpdatedTime with ...
Could you please help me in resolving this issue?

.NET version 8
Microsoft.Azure.Devices SDK version :2.0.0-preview007

**Comment by timtay-microsoft** [comment]
This sounds like a bug in the SDK. Can you provide the full error message in the [issue thread](https://github.com/Azure/azure-iot-sdk-csharp/issues/3411) I opened for you?

**Reply by chandra-rayaprol** [reply]
It seems that this SDK is unlisted and not available for adding to the Visual Studio project?

**Reply by jonekdahl** [reply]
Yeah, what has happened to the preview 2.x releases? They're no longer listed on nuget.org as far as I can see (I'm looking at the [Microsoft.Azure.Devices.Client versions](https://www.nuget.org/packages/Microsoft.Azure.Devices.Client/1.42.2#versions-body-tab)).

**Comment by chandra-rayaprol** [comment]
I am unable to view the SDK 'Microsoft.Azure.Device 2.x' preview versions in the Visual Studio NuGet Browser that were previously available. Kindly let me know the status and if there are any plans to restore them.
